src/async-wrap.cc

```diff
@@ -38,6 +38,8 @@ static void SetupHooks(const FunctionCallbackInfo<Value>& args) {
   //   non-zero to have those callbacks called. This only affects the init
   //   callback. If the init callback was called, then the pre/post callbacks
   //   will automatically be called.
+  // - kEnabled (1): Tells the AsyncWrap constructor whether it should
+  //   make any calls to the JS hook functions.
```

Comment:
Also, can you distinguish what "JS hook functions" this means? There are 4 of them, and this doesn't affect them all.